mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -4,15 +4,33 @@
 //
 //==================================================================================================
 
+// Bugs:
+// Cubemap and ambient cube can change on models based on flashlight state and direction you're looking
+
+// Includes for all shaders
 #include "BaseVSShader.h"
 #include "cpp_shader_constant_register_map.h"
 
+// Includes specific to this shader
 #include "pbr_vs20.inc"
 #include "pbr_ps30.inc"
 
+// Defining samplers
+#define SAMPLER_BASETEXTURE SHADER_SAMPLER0
+#define SAMPLER_NORMAL SHADER_SAMPLER1
+#define SAMPLER_ENVMAP SHADER_SAMPLER2
+#define SAMPLER_SHADOWDEPTH SHADER_SAMPLER4
```

Comment:
Would you mind using consts here instead of defines?

Example: `const Sampler_t SAMPLER_BASETEXTURE = SHADER_SAMPLER0;`

Reply:
Sure, I'll get back on the computer after dinner and do it

Reply:
Done